database/postgres/postgres.go

```diff
@@ -358,6 +358,38 @@ func (p *Postgres) Drop() (err error) {
 			}
 		}
 	}
+	query = `SELECT typname FROM pg_type WHERE typcategory = 'E';`
+	types, err := p.conn.QueryContext(context.Background(), query)
+	if err != nil {
+		return &database.Error{OrigErr: err, Query: []byte(query)}
+	}
+	defer func() {
+		if errClose := types.Close(); errClose != nil {
+			err = multierror.Append(err, errClose)
+		}
+	}()
+	typeNames := make([]string, 0)
+	for types.Next() {
+		var tableName string
+		if err := tables.Scan(&tableName); err != nil {
+			return err
+		}
+		if len(tableName) > 0 {
+			tableNames = append(tableNames, tableName)
+		}
+	}
+	if err := types.Err(); err != nil {
+		return &database.Error{OrigErr: err, Query: []byte(query)}
+	}
+	if len(typeNames) > 0 {
+		// delete one by one ...
+		for _, t := range tableNames {
+			query = `DROP TYPE IF EXISTS ` + pq.QuoteIdentifier(t)
```

Comment:
Why not [`CASCADE`](https://www.postgresql.org/docs/current/sql-droptype.html) the `DROP`?

Reply:
At this point there are no tables to cascade right? But I guess it doesn't hurt.